go/governance/api/api.go

```diff
@@ -125,12 +126,15 @@ func (p *ProposalContent) Equals(other *ProposalContent) bool {
 // given writer.
 func (p ProposalContent) PrettyPrint(ctx context.Context, prefix string, w io.Writer) {
 	switch {
-	case p.Upgrade != nil && p.CancelUpgrade == nil:
+	case p.Upgrade != nil:
```

Comment:
I think it's fair to assume that the caller validates the proposal before pretty printing it? (that ensures only a single field is set)

Reply:
Or, you could also print all fields if they are set, to not hide that the proposal is invalid.

Reply:
Yeah i guess